corehq/apps/hqwebapp/management/commands/resource_static.py

```diff
@@ -42,7 +42,10 @@ def handle(self, *args, **options):
             for path, storage in finder.list(['.*', '*~', '* *']):
                 if not storage.location.startswith(prefix):
                     continue
-                url = os.path.join(storage.prefix, path) if storage.prefix else path
+                if not getattr(storage, 'prefix', None):
```

Comment:
is one of these cases for backwards compatibility or does django 1.7 need both of these branches?